### ISSUE TITLE
policy condition type http-status

### ISSUE THREAD

**Issue by VincentPicardEspalion** [F5Networks/f5-appsvcs-extension#755]
### Is your feature request related to a problem? Please describe.
Actually, one of my VS has the following ltm policy conditions :
            conditions {
                0 {
                    http-status
                    response
                    code
                    greater-or-equal
                    values { 300 }
                }
                1 {
                    http-status
                    response
                    code
                    less
                    values { 400 }
                }
            }
        }
I want to code this in AS3 but I cannot find https-status type in the AS3 schema

### Describe the solution you'd like
I would like http-satus type to be available in AS3

### Describe alternatives you've considered
If there is a method to get the same conditions with the existing AS3 types, thank you for telling me

### Additional context
Add any other context, such as the desired tmsh configuration, about the feature request here.

**Comment by RobCupples** [comment]
When v3.48.0 is released you will be able to do this with this snipper

```
...
            "conditions": [
              {
                "type": "httpStatus",
                "event": "response",
                "code": {
                  "operand": "greater-or-equal",
                  "values": [
                    300
                  ]
                }
              },
              {
                "type": "httpStatus",
                "event": "response",
                "code": {
                  "operand": "less",
                  "values": [
                    400
                  ]
                }
              }
            ]
...
```

```
            conditions {
                0 {
                    http-status
                    response
                    code
                    greater-or-equal
                    values { 300 }
                }
                1 {
                    http-status
                    response
                    code
                    less
                    values { 400 }
                }
            }
```
```

**Comment by sunitharonan** [comment]
Closing as this has been released in AS3 3.48.0
